regression/tests/helpers.py

```diff
@@ -4,16 +4,23 @@
 import os
 
 
+COURSE_ORG = 'COURSE_ORG'
+COURSE_NUMBER = 'COURSE_NUMBER'
+COURSE_RUN = 'COURSE_RUN'
+COURSE_DISPLAY_NAME = 'COURSE_DISPLAY_NAME'
+
+
 def get_course_info():
     """
     Returns the course info of the course that we use for
     the regression tests.
     """
     return {
-        'org': 'ArbiRaees',
-        'number': 'AR-1000',
-        'run': 'fall',
-        'display_name': 'Manual Smoke Test Course 1 - Auto'
+        'org': os.environ.get(COURSE_ORG, 'ArbiRaees'),
```

Comment:
I think that you should have it default to None, rather than the values you want. The end-result should be that the user sets the values before running the test. If you prefer, you can store these in a separate .ini file or something along those lines.